Readme.md

```diff
@@ -1,20 +1,46 @@
+    RPGKeeper
+</h1>
 
-I am an avid table-top RPG player. I have been playing various forms of RPGs for years (currently it's mainly D&D, and 
-Edge of the Empire). Frequently, I felt limited or frustrated by the paper character sheets I had. And sometimes I'd
-lose them, or something would get spilled, or I'd want to look up something about my character, but didn't have it on me.
-Any of you who've played table-top RPGs know what I'm talking about.
+[![Total alerts](https://img.shields.io/lgtm/alerts/g/Morgul/rpgkeeper.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/Morgul/rpgkeeper/alerts/)
+[![Language grade: JavaScript](https://img.shields.io/lgtm/grade/javascript/g/Morgul/rpgkeeper.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/Morgul/rpgkeeper/context:javascript)
 
-That's why I've decided to make a website for storing your characters. It's a daunting task, and I've been working on
-various versions of it for years; this is only the latest iteration. And yes, it does keep getting better.
+* Production: https://rpgkeeper.com
+* Beta: https://beta.rpgkeeper.com
 
-My vision for RPGKeeper is that it become a site useful for groups of players as well as individuals. I want GMs and 
-Players to love it, using it for big campaigns, and as a central clearinghouse for campaign related things. At the 
-moment, most of these features are missing, but I will be working on adding them just as soon as I get my core target 
-systems built and working.
+RPGKeeper is a digital chracter manager. While there are many others out there that are targeted to one or two systems, RPGKeeper aims to be a _universal_ character management system. It can support any TTRPG system ever created, as long as that system is added into it's codebase. Building a system is a time-consuming process, but it's possible to get the basics going in just a few hours. And with support for a 'generic' system, RPGKeeper allows you to get down to playing quickly even if it doesn't support your game with a dedicated system.
+
+## FAQ
+
+### But what about Roll20?
+
+Don't get us wrong, Roll20 certainly has a lot of features, and a dedicated fanbase. But Role20 is a map first, and a character/campaign manager second. Often, character sheets are built based off the official paper sheets from games; which often don't translate well to a digital format. RPGKeeper takes a different approach: we focus on building user experiences centered around playing your character, without worrying about keeping the sheet the same as the paper version. Characters are a first class citizen.
+
+### How many Characters can I have?
+
+Currently there are no limits on number of characters.
+
+### How much does RPGKeeper cost?
+
+Currently, RPGKeeper is free. In the future, there is a plan to monetize the project, but there will always be a free tier, and you will not lose access to any existing characters once we implement that plan.
```

Comment:
If this is for developers, it’s not really relevant. Add that the previous section is about limits, and this section implies limits will be imposed in the future, it seems like it’s settings users up for disappointment.